assets/src/styles/base/_variables.scss

```diff
@@ -1,4 +1,5 @@
 // Breakpoints
-$mq-lg-up: "( min-width: 1200px )";
+$mq-lg-up: "( min-width: 1300px )";
+$mq-lg-down: "( max-width: 1300px )";
```

Comment:
Is `$mq-lg-down` still being used? If it is, both of these will trigger at 1300px exactly, I'd maybe do min 1300 max 1299.